woocommerce/woocommerce-ios
# Product Description: Update placeholder text for Aztec Editor VCs

Resolves #3667 

### Description
- Updated the placeholder text for the Product Description / Product Short Description screens
    - No title: “Describe your product to your future customers…”
    - With title: “Tell us more about [product title]…”
    
### How to test
1. Create a product
2. Without assigning the product title, tap on the product description row
3. ✅ The placeholder text should read: `Describe your product to your future customers…`
4. Go back to the previous screen
5. Enter some text for the product title, then tap on the product description row
6. ✅ The placeholder text should read: `Tell us more about [product title]…`

### Screenshots

Before | After (No title) | After (With title)
--- | --- | ---
![simulator_screenshot_348AD402-1487-451B-9B63-0C294AF00C5B](https://user-images.githubusercontent.com/6711616/108027823-21c1e200-706e-11eb-8324-f920cc5e08d5.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-02-16 at 15 33 09](https://user-images.githubusercontent.com/6711616/108027761-08b93100-706e-11eb-866d-90a7bf85031b.png) | ![Simulator Screen Shot - iPhone 12 Pro - 2021-02-16 at 15 33 19](https://user-images.githubusercontent.com/6711616/108027762-0951c780-706e-11eb-8522-7e39e5359853.png)


Update release notes:
